src/Compilers/CSharp/Portable/Binder/SubsumptionDiagnosticBuilder.cs

```diff
@@ -129,6 +129,8 @@ private ErrorCode CheckSubsumed(BoundPattern pattern, DecisionTree decisionTree,
                         // the input can no longer be null. In that case a null pattern is subsumed.
                         if (isNull && !inputCouldBeNull)
                         {
+                            // Note: we do not have any test covering this. Is it reachable?
```

Comment:
I don't think we should keep dead code just because something might get expanded in the future. If it gets expanded, the code should be added as part of the feature development process, go through standard review and testing process. 

---
In reply to: [100596825](https://github.com/dotnet/roslyn/pull/16928#discussion_r100596825) [](ancestors = 100596825,100589351)